src/amo/components/SearchResult/styles.scss

```diff
@@ -7,6 +7,10 @@ $icon-default-size: 32px;
 .SearchResult {
   list-style-type: none;
   margin: 0 0 1px;
+
+  :hover {
+    cursor: pointer;
+  }
 }
 
 .SearchResult-link {
```

Comment:
This needs some tweaks to get the h2 link looking how it did before (when everything was wrapped in a link).

```diff
diff --git a/src/amo/components/SearchResult/styles.scss b/src/amo/components/SearchResult/styles.scss
index 1f5b3813a..36c3d20c8 100644
--- a/src/amo/components/SearchResult/styles.scss
+++ b/src/amo/components/SearchResult/styles.scss
@@ -14,8 +14,19 @@ $icon-default-size: 32px;
 }
 
 .SearchResult-link {
-  color: $text-color-default;
   text-decoration: none;
+
+  &,
+  &:active,
+  &:link,
+  &:visited {
+    color: $text-color-default;
+  }
+
+  &:focus,
+  &:hover {
+    color: $link-color;
+  }
 }
 
 .SearchResult-result {
@@ -128,11 +139,6 @@ $icon-default-size: 32px;
     }
   }
 
-  .SearchResult-link:focus &,
-  .SearchResult-link:hover & {
-    color: $link-color;
-  }
-
   .LoadingText {
     margin-bottom: 5px;
   }
```

Reply:
Thanks! I also had to add:

```diff

diff --git a/src/amo/components/SearchResult/styles.scss b/src/amo/components/SearchResult/styles.scss
index 2f455eeb5..59ad7d729 100644
--- a/src/amo/components/SearchResult/styles.scss
+++ b/src/amo/components/SearchResult/styles.scss
@@ -10,6 +10,10 @@ $icon-default-size: 32px;
 
   &:hover {
     cursor: pointer;
+
+    .SearchResult-link {
+      color: $link-color;
+    }
   }
 }
```

In order to make the link change colour when the user hovers anywhere over the `li`, which is how it behaves in the current version.